java/src/jmri/ShutDownManager.java

```diff
@@ -137,5 +137,5 @@ public interface ShutDownManager extends PropertyChangeProvider {
      *
      * @return true if shutting down or restarting
      */
-    public boolean isShuttingDown();
-}
+    boolean isShuttingDown();
+}
```

Comment:
The static analysis report at https://github.com/JMRI/JMRI/actions/runs/5707799844/job/15464898754?pr=12235
has picked up that this file needs to end with a new Line.

Reply:
Hi @icklesteve.

I missed that. Im sorry. I have commited a fix.